src/AngularPublic.js

```diff
@@ -265,5 +265,6 @@ function publishExternalAPI(angular) {
         $$cookieReader: $$CookieReaderProvider
       });
     }
-  ]);
+  ])
+  .info({ angularVersion: '"NG_VERSION_FULL"' });
```

Comment:
Why not call it just `version` (since it's already namespaced by the module name).
Especially in other modules (`ngAnimate`, `ngAria`, etc).